src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2137,7 +2156,7 @@ void InitThermalAndFluxHistories(EnergyPlusData &state)
     auto &Surface(state.dataSurface->Surface);
 
     // First do the "bulk" initializations of arrays sized to NumOfZones
-    state.dataHeatBal->ZoneMRT = 23.0;       // module level array
+    state.dataHeatBal->ZoneMRT = 23.0;   // module level array
     state.dataHeatBalFanSys->MAT = 23.0; // DataHeatBalFanSys array
     state.dataHeatBalFanSys->ZT = 23.0;
```

Comment:
We should avoid using these array initializers and initialize explicitly using a loop. This should also allow us to combine these different initialization loops into a single loop.